example_1_basic_read/main.py

```diff
@@ -0,0 +1,25 @@
+"""Speakleash: basic read of metadata and documents"""
+
+import os
+
+from speakleash import Speakleash
+
+
+if __name__ == '__main__':
+
+    base_dir = os.path.join(".")
+    replicate_to = os.path.join(base_dir, "datasets")
+    print("Replicate to:", replicate_to)
```

Comment:
More common thing is to use f'string formatting like:
print(f"Replicate to: {replicate_to}")

Reply:
Missed that - the same change will be in example no. 3 (quality_metrics)